models/model_constants.py

```diff
@@ -1,8 +1,8 @@
 # model Runner actions constants
 from DataProcessing import dataProcessingConstants
 TRACKING = 'tracking'
-RE_ID = 're-id'
-
-BASIC_ACTIONS = [TRACKING, RE_ID]
+RE_ID_EVAL = 're-id-eval'
+RE_ID_TRAIN = 're-id-train'
```

Comment:
Add a list of Re-ID capacities e.g [RE_ID_EVAL, RE_ID_TRAIN , ... ] into a single list of constants